adding chapter 3 (#3)
mine-cetinkaya-rundel/r4ds-solutions: chapter-3.qmd

---
title: "Workflow: basics"
---

## Solutions

```{r}
#| label: load-packages
#| message: false

library(tidyverse)
```

## 3.5 Exercises

> 1. Because in "my_variable" theres a 1 instead of an i in variable.

Reviewer: ```suggestion
1. Because in `my_variable` theres a ı (i without a dot) instead of an i in variable name.
```

> 2.

Reviewer: This is missing the library() function and the geom_smooth() layer.

```{r}
ggplot(mpg) + 
  geom_point(aes(x = displ, y = hwy))
```

> 3. Alt + Shift + K Pulls up keyboard shortcuts.

Reviewer: ```suggestion
3. Alt + Shift + K pulls up keyboard shortcuts.
```

> 4. The "my_bar_plot" is saved because thats what was specified.

Reviewer: ```suggestion
4. The `my_bar_plot` object is saved because that's what was specified in the `plot` argument of the `ggsave()` call.
```

---
title: "Workflow: basics"
---

##Solutions

## 3.5 Exercises

1.why does this code not work?

because in my_variable theres a 1 instead of an i in variable

2.tweak each of th R commands so they run properly

```{r}
library(tidyverse)

ggplot(data = mpg) + 

>   geom_point(mapping = aes(x = displ, y = hwy))

Author: ```suggestion
  geom_point( aes(x = displ, y = hwy))
```

---
title: "Workflow: basics"
---

## Solutions

```{r}
#| label: load-packages
#| message: false

library(tidyverse)
```

## 3.5 Exercises

1.why does this code not work?

because in my_variable theres a 1 instead of an i in variable

> 2.tweak each of th R commands so they run properly

Reviewer: ```suggestion
2. Tweak each of the R commands so they run properly.
```

```{r}

> 

Reviewer: Remove this space

library(tidyverse)

ggplot(mpg) + 
  geom_point(aes(x = displ, y = hwy))

> 

Reviewer: Remove extraneous spaces.

```

3.Press Alt + Shift + K. What happens? How can you get to the same place using the menus?

> pulls up keyboard shortcuts

Reviewer: Capitalization and period at the end of sentence.
